.circleci/config.yml

```diff
@@ -304,6 +318,7 @@ jobs:
           device: iPhone 14
           version: '16.4'
       - setup-flutter
+      - disable-spm-feature
```

Comment:
Do we need to enable/disable SPM on the example project as well? Just checking since maybe the sample app is still using cocoapods always right? Not sure if we need to have a separate sample app to test using SPM though 🤔